### ISSUE TITLE
Win10系统下，loss.py报错ValueError: (InvalidArgument) Tensor holds the wrong type, it holds int, but desires to be int64_t.

### ISSUE THREAD

**Issue by davidhefan** [PaddlePaddle/PaddleNLP#322]
请帮忙看看会是什么问题？
![图像 1](https://user-images.githubusercontent.com/13231750/116493683-ce6fd980-a8d1-11eb-8d24-911d6f786471.png)

**Comment by wawltor** [comment]
您好，看错误是softmax_with_cross_entropy的输入的labels的数据类型需要是int64, 但是代码中的是int32, 所以您可以选择将数据流中labels的数据类型转为int64, 或者是在 `paddle.mean(criteion(logits, labels.reshape([-1])))` 这行代码之前将labels 进行数据类型转化，具体代码可以参考 `labels = paddle.cast(labels, dtype="int64")`

同时跟您确认一下，这份代码是LiC2021的事件抽取代码，没有经过修改对吗？

**Comment by Steffy-zxf** [comment]
@davidhefan 你好，具体修改可以参考该修复pr：https://github.com/PaddlePaddle/PaddleNLP/pull/330

**Comment by davidhefan** [comment]
@wawltor 是LiC2021的事件抽取代码，没有经过修改。只是放在本地win10运行。您给的参考是正确的解决办法。感谢。
@Steffy-zxf 感谢。
这个问题可能是什么原因导致的？liunx和windows的区别吗？我感觉应该不是系统的问题。

**Comment by Steffy-zxf** [comment]
@davidhefan 是由于windows下，numpy对于int型数据默认的是`int32`类型，而 softmax_with_cross_entropy op要求是`int64`类型，导致不匹配报错。

**Comment by davidhefan** [comment]
明白了，感谢！
